admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='creating-LC-and-ASG-cluster-auto-scaler-{context}']
+= Creating the Launch Configuration and Auto Scaling group
+
+Before you deploy the the cluster auto-scaler, you must create an Amazon Web
+Services (AWS) Launch Configuration and Auto Scaling group that reference a
+primed image. You must configure the LC so that the new node automatically joins
+the existing cluster when it starts.
+
+.Prerequisites
+
+* Install a {product-title} cluster in AWS.
+* Create a primed image.
+
+.Procedure
+
+. Create the *_bootstrap.kubeconfig_* file by copying it from the master node:
++
+[source,bash]
+----
+$ ssh master "sudo cat /etc/origin/master/bootstrap.kubeconfig" > ~/bootstrap.kubeconfig
+----
+
+. Create the *_user-data.txt_* cloud-init file from the *_bootstrap.kubeconfig_*
+file:
++
+[source,bash]
+----
+$ cat <<EOF > user-data.txt
+#cloud-config
+write_files:
+- path: /root/openshift_bootstrap/openshift_settings.yaml
+  owner: 'root:root'
+  permissions: '0640'
+  content: |
+    openshift_node_config_name: node-config-compute 
+- path: /etc/origin/node/bootstrap.kubeconfig
+  owner: 'root:root'
+  permissions: '0640'
+  encoding: b64
+  content: |
+    $(base64 ~/bootstrap.kubeconfig | sed '2,$s/^/    /')
+
+runcmd:
+- [ ansible-playbook, /root/openshift_bootstrap/bootstrap.yml]
+- [ systemctl, restart, systemd-hostnamed]
+- [ systemctl, restart, NetworkManager]
+ifdef::openshift-enterprise[]
+- [ systemctl, enable, atomic-openshift-node]
+- [ systemctl, start, atomic-openshift-node]
+endif::[]
+ifdef::openshift-origin[]
+- [ systemctl, enable, origin-node]
+- [ systemctl, start, origin-node]
+endif::[]
+EOF
+----
+
+. Create the Launch Configuration by using the AWS CLI:
```

Comment:
s/by using/using ??

Reply:
IBM Style says:

using: To avoid ambiguity, replace this gerund with either “by using” or “that uses.”